### ISSUE TITLE
Disease remission

### ISSUE THREAD

**Issue by rays22** [obophenotype/human-phenotype-ontology#7406]
### Preferred term label:

Disease remission

### Synonyms

remission

### Definition (free text, please give PubMed ID)

A decrease in or disappearance of signs and symptoms typically associated with a chronic disease course.

* Sources:
    * [Remission (medicine)](https://en.wikipedia.org/wiki/Remission_(medicine)) Remission is either the reduction or disappearance of the signs and symptoms of a disease
    * [remission](https://www.cancer.gov/publications/dictionaries/cancer-terms/def/remission) A decrease in or disappearance of signs and symptoms of cancer. In partial remission, some, but not all, signs and symptoms of cancer have disappeared. In complete remission, all signs and symptoms of cancer have disappeared, although cancer still may be in the body.
    * [NCIT:C18246 Remission](https://www.ebi.ac.uk/ols/ontologies/ncit/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FNCIT_C18246) An abatement in intensity or degree (as in the manifestations of a disease). [ NCI ]
    * [OGMS:0000106 remission](https://www.ebi.ac.uk/ols/ontologies/ogms/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FOGMS_0000106) A disease stage which is characterized by the lack of a manifestation of a disease.  [remission: a temporary convalescence typically associated with a chronic disease course](https://code.google.com/archive/p/ogms/issues/73)

### Parent term (use hpo.jax.org/app)

[Clinical course HP:0031797](https://hpo.jax.org/app/browse/term/HP:0031797)

### Diseases characterized by this term ? (e.g. Orphanet or OMIM number)

[OMIM: 601626 LEUKEMIA, ACUTE MYELOID; AML](https://www.omim.org/entry/601626?search=remission&highlight=remission)

### Your nano-attribution (ORCID)

<https://orcid.org/0000-0001-8314-2140>

**Comment by pnrobinson** [comment]
Thanks, added as HP_0034382
removed 'chronic' from definition:
A decrease in or disappearance of signs and symptoms typically associated with a disease course.
